mat-chalmers.cabal

```diff
@@ -1,3 +1,4 @@
+cabal-version: 3.0
```

Comment:
This PR doesn't build on my machine, with this error: 

```
/home/hace/dev/haskell/mat-chalmers/src/View.hs:33:26: error:
    Variable not in scope:
      defaultTimeLocale :: System.Locale.TimeLocale
   |
33 |       toHtml (formatTime defaultTimeLocale "%F" date)
   |                          ^^^^^^^^^^^^^^^^^
```

Probably because some version change. 

- [ ] Could you set some version interval for the packages, instead of having none? Having a version interval should make it easier to build the project using stack or cabal without nix. I'm using stack which is why I have an interest in this.

Reply:
Can you test now? I tried using a remote machine to compile

Reply:
That one works, but now it complains about running on ghc947. :D

Reply:
Double check this?

Reply:
I think it works now.

Reply:
Still building

Reply:
Build complete, everything good!